src/clf/compoundV3/compoundV3Computer.js

```diff
@@ -0,0 +1,305 @@
+const DATA_DIR = process.cwd() + '/data';
+const spans = [7, 30, 180];
+
+async function compoundV3Computer(fetchEveryMinutes) {
+    const start = Date.now();
+    try {
+        await RecordMonitoring({
+            'name': 'CompoundV3 CLF Computer',
+            'status': 'running',
+            'lastStart': Math.round(start / 1000),
+            'runEvery': fetchEveryMinutes * 60
+        });
+        if (!process.env.RPC_URL) {
+            throw new Error('Could not find RPC_URL env variable');
+        }
+
+        if (!fs.existsSync(`${DATA_DIR}/clf/`)) {
+            fs.mkdirSync(`${DATA_DIR}/clf/`);
+        }
+
+        console.log(`${fnName()}: starting`);
+        const web3Provider = new ethers.providers.StaticJsonRpcProvider(process.env.RPC_URL);
+        const currentBlock = await web3Provider.getBlockNumber() - 10;
+
+        const results = {};
+        /// for all pools in compound v3
+        for (const pool of Object.values(compoundV3Pools)) {
+            results[pool.baseAsset] = await computeCLFForPool(pool);
+            const poolData = computeAverageCLFForPool(results[pool.baseAsset]);
+            results[pool.baseAsset]['weightedCLF'] = poolData['weightedCLF'];
+            results[pool.baseAsset]['totalCollateral'] = poolData['totalCollateral'];
+            console.log(`results[${pool.baseAsset}]`, results[pool.baseAsset]);
+        }
+
+        let protocolWeightedCLF = undefined;
+        try {
+            protocolWeightedCLF = computeProtocolWeightedCLF(results);
+        }
+        catch (error) {
+            console.log(error);
+        }
+        const toRecord = {
+            protocol: 'compound v3',
+            weightedCLF: protocolWeightedCLF ? protocolWeightedCLF : undefined,
+            results
+        };
+
+
+
+        recordResults(toRecord);
+
+        console.log('CompoundV3 CLF Computer: ending');
+
+        const runEndDate = Math.round(Date.now() / 1000);
+        await RecordMonitoring({
+            'name': 'CompoundV3 CLF Computer',
+            'status': 'success',
+            'lastEnd': runEndDate,
+            'lastDuration': runEndDate - Math.round(start / 1000),
+            'lastBlockFetched': currentBlock
+        });
+    } catch (error) {
+        const errorMsg = `An exception occurred: ${error}`;
+        console.log(errorMsg);
+        await RecordMonitoring({
+            'name': 'CompoundV3 CLF Computer',
+            'status': 'error',
+            'error': errorMsg
+        });
+    }
+
+}
+
+function recordResults(results) {
+    const date = getDay();
+    if (!fs.existsSync(`${DATA_DIR}/clf/${date}`)) {
+        fs.mkdirSync(`${DATA_DIR}/clf/${date}`);
+    }
+    if (!fs.existsSync(`${DATA_DIR}/clf/latest`)) {
+        fs.mkdirSync(`${DATA_DIR}/clf/latest`);
+    }
+    const unifiedFullFilename = path.join(DATA_DIR, `clf/${date}/${date}_compoundV3_CLFs.json`);
+    const latestFullFilename = path.join(DATA_DIR, 'clf/latest/compoundV3_CLFs.json');
```

Comment:
the first variable name is wrong, I think it comes from a copy/paste from my code :)

should be named "datedFullFilename" I think